runtimes/eoapi/stac/eoapi/stac/app.py

```diff
@@ -58,115 +54,78 @@
 )
 templates = Jinja2Templates(env=jinja2_env)
 
-api_settings = ApiSettings()
+settings = Settings()
 auth_settings = OpenIdConnectSettings()
-settings = api_settings.load_postgres_settings()
 
-enabled_extensions = api_settings.extensions or []
 
 # Logs
-init_logging(debug=api_settings.debug)
+init_logging(debug=settings.debug)
 logger = logging.getLogger(__name__)
 
 # Extensions
 # application extensions
-application_extensions_map = {
-    "transaction": TransactionExtension(
-        client=TransactionsClient(),
-        settings=settings,
-        response_class=ORJSONResponse,
-    ),
-    "bulk_transactions": BulkTransactionExtension(client=BulkTransactionsClient()),
-}
```

Comment:
removed the `transaction` extensions because I assume we will:
- either add the ingestor api 
- add them back once we have some auth defined